libs/api/domains/driving-license/src/lib/drivingLicense.spec.ts

```diff
@@ -393,6 +393,7 @@ describe('DrivingLicenseService', () => {
         needsToPresentHealthCertificate: false,
         needsToPresentQualityPhoto: false,
         licenseCategory: DrivingLicenseCategory.B,
+        sendLicenseInMail: 0,
```

Comment:
I see we're using numbers and booleans for sendLicenseInMail.

Technically it is mostly ok to mix `0` and `false`, `1` and `true`.
But why not just use `true` and `false` or just numbers?

Strict comparison will fail though:
```javascript
> Boolean(0)
false
> Boolean(1)
true
> true == 1
true
> true === 1
false
```